### DISCUSSION TITLE
添加github账号切换功能(Add GitHub account switching function)

### DISCUSSION THREAD

**Discussion by cutelolly** [community/community#50544] [Mobile]
对于多用户可以添加账号切换功能。

**Comment by aconsuegra** [comment]
Hi @cutelolly, thanks for the feedback! 

If I have understood your request properly, I believe you are asking for multi-account support in the apps. The GitHub Mobile apps currently support multi-account for one github.com and one [GHES](https://docs.github.com/en/enterprise-server@3.8/admin/overview/about-github-enterprise-server) account. Support for more that one github.com is currently unsupported by our Terms of Service, but we hope to have some good news in this topic soon.

I will close this discussion as duplicated of https://github.com/orgs/community/discussions/3337, please feel free to upvote that one.
